docs-gb/getting-started/configuration.md

```diff
@@ -26,10 +26,22 @@ The `consumerGroupIdPrefix` will ensure that all consumer groups created have a
 
 ### Kubernetes
 
-For Kubernetes this is controlled via a ConfigMap called `seldon-kafka` whose default values are
-defined in the `SeldonConfig` custom resource.
-
-{% @github-files/github-code-block url="https://github.com/SeldonIO/seldon-core/blob/v2/k8s/yaml/components.yaml" %}
```

Comment:
Not specifically for this change, but please note you can reference a line-range within a given file, like this:

`https://github.com/SeldonIO/seldon-core/blob/v2/k8s/yaml/components.yaml#L20-L47`

Reply:
line range not rendering in GitBook.

Reply:
It appears ok when I add it directly via the gitbook app. See [here](https://app.gitbook.com/o/rOXFWV2k010iw8QdOyaH/s/n8ONqKsfp9k7WHp5OWSM/~/changes/81/getting-started/configuration). Does this then not show up when rendering the actual page? That would be a bug that we should report.

Reply:
Hi @lc525  I checked the preview and it renders fine. https://docs.seldon.ai/seldon-core-2/getting-started/configuration